packages/cli/test/services/project-checker.test.ts

```diff
@@ -54,4 +61,171 @@ describe('project checker', (): void => {
             expect(exceptionThrown).to.be.equal(true)
         })
     })
+
+    describe('checkCurrentDirBoosterVersion', (): void => {
+        
+        beforeEach(() => {
+            replace(projectUpdater,'updatePackageJsonDependencyVersions', fake.resolves({}))
+            replace(logger,'info', fake.resolves({}))
+        })
+
+        describe('inside a Booster project', () => {
+            //project version in mocked package.json is 1.11.2
+            beforeEach(() => {
+                replace(process,'cwd', fake.returns(path.join(process.cwd(),'test', 'fixtures', 'mock_project')))
```

Comment:
```suggestion
                replace(process, 'cwd', fake.returns(path.join(process.cwd(), 'test', 'fixtures', 'mock_project')))
```

By the way, don't worry too much about these, I'm just worried about the linter failing :v: